### PR TITLE
Ensure tuples. Dont assume that it is already

### PR DESCRIPTION
**Pull request by alanjds**
Some strange people (like me) use lists instead of tuples. Guard against this kind of people.

### CODE REVIEW

File: denorm/denorms.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def get_alldenorms():
     alldenorms = []
     for model in gmodels.get_models(include_auto_created=True):
         if not model._meta.proxy:
-            for field in model._meta.fields + model._meta.local_many_to_many:
+            for field in tuple(model._meta.fields) + tuple(model._meta.local_many_to_many):
```
Comment (alexmadjar) [inline denorm/denorms.py:77]: Couple of questions:
1.  `model._meta` is a Django API so I don't see how you even have control over the type here
2.  lists should also sum correctly

Comment (alanjds) [inline denorm/denorms.py:77]: 1. You usually control this via the `Meta` class created into the models to change its behaviours. It becomes `_meta` later. So, the `Meta.fields` you put there changes to `_meta.initial_fields`, if I remember correctly.
2. The thing is a tuple by default. However, is usual to set the `Meta.fields` as a list and not a tuple, to ease dynamic customisation on the `__init__` of forms and admins. But then it does not sum anymore if left and right are of different types (list and tuple).

**Comment by alexmadjar** [comment]
okay... why not!

**Comment by alexmadjar** [comment]
@alanjds - created tag `v0.3.1` for you

**Comment by alanjds** [comment]
Thanks.

**Pull request merged**